src/agent/Cargo.toml

```diff
@@ -22,6 +22,12 @@ scan_fmt = "0.2.3"
 scopeguard = "1.0.0"
 regex = "1"
 
+async-trait = "0.1.42"
+tokio = { version = "0.2", features = ["rt-core", "sync", "uds", "stream", "macros", "io-util", "time", "signal", "io-std", "process", "blocking"] }
```

Comment:
This is an old version - the latest is `"1.0.1"`, and the feature names have also changed fwics.

Reply:
I plan to merge this pr first and then upgrade tokio in a new pr, because many dependent libraries need to be upgraded, which takes a lot of time. I think we can introduce the async runtime first and then we enhance it.

Reply:
I was looking on what it takes to use latest version of Tokio. tokio-vsock and ttrpc needs to be upgraded to 1.0 first.

Reply:
Yes, I have already created an issue for `tokio-vsock` and will update ttrpc in days.

Reply:
and the netlink-sys also needs to be upgraded

Reply:
If we can't update on this PR, please could one of you open a tracking issue so we don't forget (and ideally reference that GH issue URL in the `Cargo.toml` :smile:

Reply:
@mxpv The pr for `tokio-vsock` to upgrade `tokio` https://github.com/fsyncd/tokio-vsock/pull/5

Reply:
> many dependent libraries need to be upgraded

I've opened a tracking issue: https://github.com/kata-containers/kata-containers/issues/1257

Reply:
@mxpv ok, thank you